packages/contract/README.md

```diff
@@ -80,51 +79,47 @@ v4.0.2 or higher.
 
 ```javascript
+const contract = require("@truffle/contract");
 // Require the package that was previosly saved by @truffle/artifactor
-var MetaCoin = require("./path/to/MetaCoin.json");
+const metacoinArtifact = require("./path/to/MetaCoin.json");
+const metacoin = contract(metacoinArtifact)
 
 // Remember to set the Web3 provider (see above).
-MetaCoin.setProvider(provider);
+metacoin.setProvider(provider);
 
 // In this scenario, two users will send MetaCoin back and forth, showing
 // how @truffle/contract allows for easy control flow.
-var account_one = "5b42bd01ff...";
-var account_two = "e1fd0d4a52...";
+const account_one = "5b42bd01ff...";
+const account_two = "e1fd0d4a52...";
 
 // Note our MetaCoin contract exists at a specific address.
-var contract_address = "8e2e2cf785...";
-var coin;
+const contract_address = "8e2e2cf785...";
 
-MetaCoin.at(contract_address).then(function(instance) {
-  coin = instance;
+const coin = await metacoin.at(contract_address)
 
+try {
   // Make a transaction that calls the function `sendCoin`, sending 3 MetaCoin
   // to the account listed as account_two.
-  return coin.sendCoin(account_two, 3, {from: account_one});
-}).then(function(result) {
+  let result = await coin.sendCoin(account_two, 3, { from: account_one });
   // This code block will not be executed until @truffle/contract has verified
   // the transaction has been processed and it is included in a mined block.
   // @truffle/contract will error if the transaction hasn't been processed in 120 seconds.
 
   // Since we're using promises, we can return a promise for a call that will
   // check account two's balance.
-  return coin.balances.call(account_two);
-}).then(function(balance_of_account_two) {
-  alert("Balance of account two is " + balance_of_account_two + "!"); // => 3
+  let balance_of_account_two = await coin.balances.call(account_two);
+  console.log("Balance of account two is " + balance_of_account_two + "!"); // => 3
 
   // But maybe too much was sent. Let's send some back.
   // Like before, will create a transaction that returns a promise, where
   // the callback won't be executed until the transaction has been processed.
-  return coin.sendCoin(account_one, 1.5, {from: account_two});
-}).then(function(result) {
-  // Again, get the balance of account two
-  return coin.balances.call(account_two)
-}).then(function(balance_of_account_two) {
-  alert("Balance of account two is " + balance_of_account_two + "!") // => 1.5
-}).catch(function(err) {
+  result = await coin.sendCoin(account_one, 1.5, { from: account_two });
+  balance_of_account_two = await coin.balances.call(account_two);
+  console.log("Balance of account two is " + balance_of_account_two + "!"); // => 1.5
+} catch(function (err) {
   // Easily catch all errors along the whole execution.
   alert("ERROR! " + err.message);
-});
+};
```

Comment:
Decided to wrap this in a try catch to keep this error handling.

Reply:
Syntax is off here -- you're got `catch(function (err) {` instead of `catch (err) {`